src/pybind/mgr/crash/module.py

```diff
@@ -45,13 +45,14 @@ def timestamp_filter(self, f):
         :param f: f(time) return true to keep crash report
         :returns: crash reports for which f(time) returns true
         """
-        def inner((_, meta)):
+        def inner(pair):
```

Comment:
nit,
why not put
```py
def inner(_, meta):
```

Reply:
because only a single argument of type `tuple` is passed to the `inner` function.

Reply:
ahh, gotcha!